internal/wazeroir/operations.go

```diff
@@ -943,7 +937,25 @@ type UnionOperation struct {
 }
 
 // String implements fmt.Stringer.
-func (o UnionOperation) String() string { return o.Kind().String() }
+func (o UnionOperation) String() string {
+	switch o.OpKind {
+	case OperationKindCall,
+		OperationKindGlobalGet,
+		OperationKindGlobalSet:
+		return fmt.Sprintf("%s %d", o.Kind(), o.B1)
+
+	case OperationKindConstI32,
+		OperationKindConstI64:
+		return fmt.Sprintf("%s %#x", o.Kind(), o.U1)
+
+	case OperationKindConstF32:
+		return fmt.Sprintf("%s %f", o.Kind(), math.Float32frombits(uint32(o.U1)))
+	case OperationKindConstF64:
+		return fmt.Sprintf("%s %f", o.Kind(), math.Float64frombits(o.U1))
+	default:
+		return o.Kind().String()
```

Comment:
can we put this `panic("TODO")` so that we won't miss the implementation of String on all kinds?

Let's add this test later similar to https://github.com/tetratelabs/wazero/blob/main/internal/wazeroir/operations_test.go#L9-L14